packages/js/tests/types/types.test.ts

```diff
@@ -26,7 +26,7 @@
  */
 
 import type { ESLint, Linter } from "eslint";
-import js from "../../";
+import js = require("@eslint/js");
```

Comment:
can we just use `import`?
```suggestion
import js from "@eslint/js";
```

Reply:
We definitely could, I used the `require` syntax since `@eslint/js` is a CJS module and I wanted to make sure the types would work fine in a CJS file. I could add a small `types.test.mts` file to test the types in an ESM setting as well. For reference I did something similar with `eslint-config-eslint` in #19050. It couldn't hurt but it's up to you guys.